NSS-Day-Cohort-42/rare-API-media-moguls
# Can now create and authenticate users.

#### Changes Made
1. Added file `auth.py` to `views` directory.
1. Modified file `urls.py`.
​
#### Steps to Review
1. Checkout this branch locally and run the application.
```
git fetch --all
git checkout mp-create-post
```
2. Run the server.
```
python manage.py runserver
```
5. Test app functionality in postman.
> Create a POST request to the URL `http://localhost:8000/register`
> Include the following body:
```
{
    "first_name": "testfirst",
    "last_name": "testlast",
    "username": "testuser",
    "email": "test@user.com",
    "password": "testing",
    "bio": "testbio",
    "profile_image_url": "testimage"
}
```
6. View code file.
> Confirm file modifications are present as indicated above.
> Confirm no unused code or extraneous comments exist.
> Confirm SQL database has been updated with the new user info.
